testsuite/features/secondary/allcli_action_chain.feature

```diff
@@ -86,13 +87,13 @@ Feature: Action chains on several systems at once
 
   Scenario: Verify action chain for two systems
     Given I am on the Systems overview page of this "sle_minion"
-    When I follow "Schedule"
+    And I follow "Schedule"
```

Comment:
"And" here would mean that it's a repeated "Given". I think "When" suits better.